### PR TITLE
New device: FORIA F00YK04-18-1

### PR DESCRIPTION
**Pull request by dinamik1488**
Add please. [https://github.com/Koenkk/zigbee2mqtt/issues/21979](url)
foria18

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -697,6 +697,46 @@ const definitions: Definition[] = [
...
+        vendor: 'FORIA',
+        description: '18 scenes remote',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [e.battery(), e.battery_voltage(), e.action(['scene_1', 'scene_2', 'scene_3', 'scene_4', 'scene_5', 'scene_6',
+            'scene_7', 'scene_8', 'scene_9', 'scene_10', 'scene_11', 'scene_12', 'scene_13', 'scene_14', 'scene_15', 'scene_16',
+            'scene_17', 'scene_18']),
+        ],
+        endpoint: (device) => {
```
Comment (Koenkk) [inline src/devices/tuya.ts:712]: I don't think these are needed, does it work without the whole `endpoint`?

Comment (dinamik1488) [reply src/devices/tuya.ts:712]: Without endpoint, it does not work, I checked it out

Comment (dinamik1488) [reply src/devices/tuya.ts:712]: yes, its ok

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -697,6 +697,43 @@ const definitions: Definition[] = [
...
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_dhke3p9w']),
+        model: 'F00YK04-18-1',
+        vendor: 'FORIA',
+        description: '18 scenes remote',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [e.battery(), e.battery_voltage(), e.action(['scene_1', 'scene_2', 'scene_3', 'scene_4', 'scene_5', 'scene_6',
```
Comment (Koenkk) [inline src/devices/tuya.ts:708]: Does the battery and battery voltage work? I don't see it in `tuyaDatapoints`.

Comment (dinamik1488) [reply src/devices/tuya.ts:708]: The battery and voltage are not working yet

Comment (Koenkk) [reply src/devices/tuya.ts:708]: Then please remove them for now.

Comment (dinamik1488) [reply src/devices/tuya.ts]: It's done

Comment (dinamik1488) [reply src/devices/tuya.ts]: It's done

**Comment by Koenkk** [comment]
I think something went wrong with the merge, see https://github.com/Koenkk/zigbee-herdsman-converters/pull/7299/files

**Comment by dinamik1488** [comment]
its done

**Pull request closed without merging**